src/camera.py

```diff
@@ -0,0 +1,181 @@
+"""
+camera
+
+Author: Steven Dropiewski
+"""
+import numpy as np
+import time
+
+from picar import filedb
+from picar.SunFounder_PCA9685 import Servo
+import picar
+
+
+class Camera:
+    """
+    Creates a camera object that can be panned and tilted.
+
+    The camera can pan in [0, 180] degrees and can tilt in [0, 150] degrees.
+
+    :param bus_number: The bus number of the servos.
+    :type bus_number: int
+
+    :param pan_channel: Pan servo's I2C channel.
+    :type pan_channel: int
+
+    :param tilt_channel: Tilt servo's I2C channel.
+    :type tilt_channel: int
+
+    :param offset: Servo's offset.
+    :type offset: int
+    """
+
+    MAX_PAN_DEGREE = 180
+    """The maximum value to be written to pan servo."""
+    MAX_TILT_DEGREE = 150
+    """The maximum value to be written to tilt servo."""
+    MIN_DEGREE = 0
+    """The minimum value to be written to both pan and tilt servo."""
+    MAX_PAN_CALI_DEGREE = 180
+    """The maximum value to calibrate pan."""
+    MAX_TILT_CALI_DEGREE = 150
+    """The maximum value to calibrate tilt."""
+    MIN_PAN_CALI_DEGREE = -180
+    """The minimum value to calibrate pan."""
+    MIN_TILT_CALI_DEGREE = -150
+    """The minimum value to calibrate tilt."""
+
+    def __init__(self, bus_number=1, pan_channel = 1, tilt_channel = 0, offset = 0):
```

Comment:
Shouldn't tilt channel be 2?

Reply:
Offset shouldn't be a parameter. That would imply that the user can change it, which would break this because we depend on having a 0 offset (on SunFounder's side anyway). See the db_file in leader/follower to for reference.